### ISSUE TITLE
line 434 probable parenthesis miss placed

### ISSUE THREAD

**Issue by Creeall** [krzysu/flot.tooltip#145]
`typeof (typeof (item.series.data[0][1] !== 'undefined'))`

should probably be 

`typeof (item.series.data[0][1]) !== 'undefined')`

**Comment by krzysu** [comment]
I don't know what are you talking about, I don't see anything like that in the code, main source file is here https://github.com/krzysu/flot.tooltip/blob/master/js/jquery.flot.tooltip.source.js

**Comment by Creeall** [comment]
If everything here is fine for you. Then forget it.

line 423
```
// replace %n with number of items represented by slice in pie charts
	if (item.series.hasOwnProperty('pie')) {
	    if (typeof (item.series.data[0][1] !== 'undefined')) {   <-----
		n = item.series.data[0][1];
	    }
	}
```

**Comment by krzysu** [comment]
oh true, you are right (!), I'm sorry, I didn't get it in the first place, I was looking for `typeof(typeof)`
but ok, I suppose it should be 
```
if (typeof item.series.data[0][1] !== 'undefined')
```
I will fix it and test, thank you!

**Comment by Creeall** [comment]
I've spent some times in this part of the code and I noticed this. Whithout any evidences of an issue in the charts themselves thought. I also could not use the :
item.series.hasOwnProperty('pie')
For some reasons, no idea yet.

I made a small change so i can send a custom tooltip content along with the server data. Instead on using build in tooltip formatter.

**Comment by krzysu** [comment]
maybe it's related to pie plugin? anyway I fixed that line in 0.9.0 release
